### ISSUE TITLE
Add pytest for utils

### ISSUE THREAD

**Issue by kkuehler** [ocf/slackbridge#50]
Since the slackbridge is arguably becoming one of the most important internal ocf projects we should add unit tests for utils and, where possible, move more functionality that can be tested into utils.

**Comment by abizer** [comment]
perhaps we can also generalize the slackbridge and offer it to the wider community for people with similar setups as us.

**Comment by ethanhs** [comment]
>perhaps we can also generalize the slackbridge and offer it to the wider community for people with similar setups as us.

I think that deserves a separate issue, or ten.

>We can also add one of those fancy coverage buttons to the project too.

Research has found that coverage testing isn't particularly effective at finding bugs. So while it has a nice badge, it would be much more effective to add type checking. Mypy has a fancy badge too now :)

**Comment by kkuehler** [comment]
@ethanhs I agree. And while I definitely think any future OCF project (or even new python scripts in puppet) could benefit from type checking, I don't really feel like doing this for slackbridge.

**Comment by dkess** [comment]
I'm thinking of closing this issue, since we do now have type checking in slackbridge, and I am skeptical that we would be able to write good unit tests for this. Anyone else have thoughts?

**Comment by keur** [comment]
Types are orthogonal to unit tests. The unit tests would just be for the utils file, which we would be able to write good tests for.  We can easily write tests for irc message formatting, nick stripping, etc.
